data/death_match_locations.json

```diff
@@ -26,6 +26,34 @@
         ],
         "boundaries": [ 2110.8208, 2055.1360, 2200.4045, 2144.6980 ]
     },
+    {
+        "id": 6,
```

Comment:
I think we'll want to keep this ordered on ID?

Reply:
Sure, I grouped them by area. Updated.